pkg/ipam/allocator/alibabacloud/alibabacloud.go

```diff
@@ -63,6 +63,14 @@ func (a *AllocatorAlibabaCloud) Init(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	// Send API requests to "vpc" network endpoints instead of the default "public" network
```

Comment:
also it is recommand to set Scheme to HTTPS . see github.com/aliyun/alibaba-cloud-sdk-go/sdk/config.go